misc/fuel/factor-mode.el

```diff
@@ -317,16 +306,15 @@ these lines in your .emacs:
 (defconst factor-symbols-lines-regex
   (concat (syntax-begin '("SYMBOLS")) ws+ symbols-to-semicolon))
 
-(defconst factor-int-constant-def-regex
-  (factor-second-word-regex '("ALIEN" "CHAR" "NAN")))
+;; (defconst factor-int-constant-def-regex
```

Comment:
Yes, mostly. But it's so tedious writing `factor-` prefixes everywhere so I thought this could be some exceptions.